src/easylink/configuration.py

```diff
@@ -89,6 +89,8 @@ def __init__(
         self,
         config_params: dict[str, Any],
         schema_name: str = "main",
+        images_dir: str | Path | None = None,
+        command: str = "run",
```

Comment:
Hack: this default is just to make tests work easily (because many of our tests instantiate a Config and I didn't want to have to add some command that isn't even used)